ossdbtoolsservice/edit_data/smo_edit_table_metadata_factory.py

```diff
@@ -6,20 +6,20 @@
 
 from typing import List  # noqa
 
-from pgsmo import Server
-from pgsmo.objects.table.table import Table  # noqa
-from pgsmo.objects.table_objects.column import Column  # noqa
 from ossdbtoolsservice.driver import ServerConnection
 from ossdbtoolsservice.edit_data import EditTableMetadata, EditColumnMetadata
 from ossdbtoolsservice.metadata.contracts.object_metadata import ObjectMetadata
 from ossdbtoolsservice.query.contracts import DbColumn
+from pgsmo import PGServer
+from pgsmo.objects.table.table import Table  # noqa
+from pgsmo.objects.table_objects.column import Column  # noqa
 
 
 class SmoEditTableMetadataFactory:
 
     def get(self, connection: ServerConnection, schema_name: str, object_name: str, object_type: str) -> EditTableMetadata:
 
```

Comment:
This looks like a common class between PG and Mysql, why are we chaning this to PGServer?

Reply:
This file originally used the pgsmo server, so I didn't want to change the functionality. But now that I'm looking at it, I think I should just go ahead and switch to the new server class!